nemo/collections/asr/parts/submodules/fast_rnnt_greedy_decoding.py

```diff
@@ -0,0 +1,440 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nemo.core.utils.cuda_python_utils import check_cuda_python_cuda_graphs_conditional_nodes_supported
+
+check_cuda_python_cuda_graphs_conditional_nodes_supported()
```

Comment:
Since this check raises an error of cuda python is not correct version, please move this check to the init of the RNNTGreedyDecodeFast